helpers.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Datadog API for Go
+ *
+ * Please see the included LICENSE file for licensing information.
+ *
+ * Copyright 2016 by authors and contributors.
+ */
+
+package datadog
+
+import "encoding/json"
+
+// Bool is a helper routine that allocates a new bool value
+// to store v and returns a pointer to it.
+func Bool(v bool) *bool { return &v }
```

Comment:
Note to self: out of curiosity, does the compiler inline this and go straight to heap? Or does this get passed in on the stack, then heap allocate, then copy, then return pointer? /curiousminds